openslides/core/views.py

```diff
@@ -447,6 +461,68 @@ def set_scroll(self, request, pk):
             scroll=request.data)
         return Response({'detail': message})
 
+    @detail_route(methods=['post'])
+    def control_blank(self, request, pk):
+        """
+        REST API operation to blank the projector.
+
+        It expects a POST request to
+        /rest/core/projector/<pk>/control_blank/ with a value for blank.
+        """
+        if not isinstance(request.data, bool):
+            raise ValidationError({'detail': 'Data must be a bool.'})
+
+        projector_instance = self.get_object()
+        projector_instance.blank = request.data
+        projector_instance.save()
+        message = "Setting 'blank' to {blank} was successful.".format(
+            blank=request.data)
+        return Response({'detail': message})
+
+    @detail_route(methods=['post'])
+    def broadcast(self, request, pk):
+        """
+        REST API operation to (un-)broadcast the given projector.
+        This method takes care, that all other projectors get the new requirements.
+
+        It expects a POST request to
+        /rest/core/projector/<pk>/broadcast/ without an argument
+        """
+        if config['projector_broadcast'] == 0:
+            config['projector_broadcast'] = pk
+            projector_broadcast_data(pk)
+            message = "Setting projector {id} as broadcast projector was successful.".format(
+                id=pk)
+        else:
+            config['projector_broadcast'] = 0
+            message = "Disabling broadcast was successful."
+        return Response({'detail': message})
+
+    @detail_route(methods=['post'])
+    def set_projectiondefault(self, request, pk):
+        """
+        REST API operation to set a projectiondefault to the requested projector. The argument
+        has to be an int representing the pk from the projectiondefault to be set.
+
+        It expects a POST request to
+        /rest/core/projector/<pk>/set_projectiondefault/ with the projectiondefault id as the argument
+        """
+        if not isinstance(request.data, int):
+            raise ValidationError({'detail': 'Data must be an int.'})
+
+        try:
+            projectiondefault = ProjectionDefault.objects.get(pk=request.data)
+            projector_instance = self.get_object()
+            projectiondefault.projector = projector_instance
+            projectiondefault.save()
+        except ObjectDoesNotExist:
```

Comment:
Please use try except else and put only the statement that may raise the exception into the try block.
Then use ProjectionDefault.DoesNotExist as exception.